service/auth_test.go

```diff
@@ -25,201 +25,186 @@
 
 package service
 
-import (
```

Comment:
Please don't move it as part of this PR. I am any way working on resolving left over issues from the JWT/JWK. Please focus on the database path :)

Reply:
It is/was database related because the idea was to attach DB(s) to services. But I try to bring it back, yes. I let you know

Reply:
It's back. Wasn't that hard